File: Wrap Up/JavaScript/GreatestCommonDivisor.js
const prompt = require('prompt-sync')();

let firstNumber = Number(prompt("Enter first number: "));
let secondNumber = Number(prompt("Enter second number: "));

let gcd = 1;
for (let count = 1; count <= firstNumber && count <= secondNumber; count++) {
    if (firstNumber % count === 0 && secondNumber % count === 0) {
        gcd = count;
    }
}
console.log("GCD: " + gcd);


//npm install prompt-sync

File: Wrap Up/Java/GreatestCommonDivisor.java
import java.util.Scanner;

public class GreatestCommonDivisor {

    public static int findgreatestCommonDivisor(int number1, int number2) {
        int greatestCommonDivisor = 1;

        for (int count = 1; count <= number1 && count <= number2; count++) {
            if (number1 % count == 0 && number2 % count == 0) {
                greatestCommonDivisor = count; 
            }
        }

        return greatestCommonDivisor;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int firstNumber = scanner.nextInt();

        System.out.print("Enter second number: ");
        int secondNumber = scanner.nextInt();

        int answer = findgreatestCommonDivisor(firstNumber, secondNumber);
        System.out.println("The GCD of " + firstNumber + " and " + secondNumber + " is: " + answer);
    }
}
